studio/components/layouts/SettingsLayout/SettingsLayout.tsx

```diff
@@ -18,7 +18,13 @@ const SettingsLayout: FC<Props> = ({ title, children }) => {
   const projectBaseInfo = ui.selectedProjectBaseInfo
 
   const router = useRouter()
-  const page = router.pathname.split('/')[4]
+  let page
```

Comment:
can we use `const` here? just a best practice to reduce the usage of `let` where we can - immutability

```
const page = router.pathname.includes('billing')
    ? router.pathname.split('/')[5]
    : router.pathname.split('/')[4]
```

Reply:
HAH Wrote that late — what was I thinking there!